From 6227c1493cc5a03f3ddb747d54b59e35c17d30b1 Mon Sep 17 00:00:00 2001
From: Sanyog Kale <skale@habana.ai>
Date: Mon, 05 Sep 2022 13:45:53 +0300
Subject: [PATCH]  [SW-102303] gaudi3: remove resolved todo's

This patch removes TODO's which are resolved as part of
functional code review.

Change-Id: Ia77b07cee7b52b47053f3c11e76110fdb1433f09
---

diff --git a/arch/gaudi3/arc/common.c b/arch/gaudi3/arc/common.c
index c10ea9c..2142a7e 100644
--- a/arch/gaudi3/arc/common.c
+++ b/arch/gaudi3/arc/common.c
@@ -257,7 +257,7 @@
 
 void enable_flr_handling_fsm(void)
 {
-	/* TODO: Check any HW FSM bits needs to be configured */
+	/* No HW FSM bits in Gaudi3, No need to set anything */
 }
 
 void pcie_phy_set_sram_ext_ld_done(void)
@@ -326,10 +326,6 @@
 void pcie_phy_ssc_en_set(void)
 {
 	if (get_ssc_en_from_strap()) {
-		/*
-		 * TODO: check whether efuse
-		 * EFUSE_SYSCFG_WA_SNPS_CASE_01038891 needs to be checked.
-		 */
 		pcie_phy_set_mpll_a_b_ssc_enable();
 	}
 }
diff --git a/arch/gaudi3/arc/dev_efuse.h b/arch/gaudi3/arc/dev_efuse.h
index 32b1378..a0e1828 100644
--- a/arch/gaudi3/arc/dev_efuse.h
+++ b/arch/gaudi3/arc/dev_efuse.h
@@ -25,7 +25,6 @@
 #define EFUSE_SYSCFG_SECURITY_ENABLE_MASK		BIT(15)
 #define EFUSE_SYSCFG_RR_MASK				BIT(16)
 #define EFUSE_SYSCFG_ENABLE_TPM_MASK			BIT(17)
- /* TODO: confirm bit for OOB protocol */
 #define EFUSE_SYSCFG_OOB_PROTOCOL			BIT(19)
 #define EFUSE_SYSCFG_BACKUP_DISABLE			BIT(22)
 #define EFUSE_SYSCFG_SKU_MASK				BIT(26)
diff --git a/arch/gaudi3/arc/recovery.c b/arch/gaudi3/arc/recovery.c
index fea0132..4fc74ee 100644
--- a/arch/gaudi3/arc/recovery.c
+++ b/arch/gaudi3/arc/recovery.c
@@ -113,16 +113,12 @@
 	arcboot_rom_status_update_bringup(img_sts_base, PROGRESS_EROM_IMG_FINISH);
 }
 
-/* TODO: remove below macros when added in specs/gaudi3 */
-#define D0_PSOC_BOOT_CONF_D0_PIF_ISO		0xC493750
-#define D0_PSOC_BOOT_CONF_D1_PIF_ISO		0xC493754
-
 static void disable_pif_isolation(void)
 {
 	if (arcboot_rom_global.die_id)
-		regmap_writel(0x0, D0_PSOC_BOOT_CONF_D1_PIF_ISO);
+		regmap_writel(0x0, mmD0_PSOC_BOOT_CONF_D1_PIF_ISO);
 	else
-		regmap_writel(0x0, D0_PSOC_BOOT_CONF_D0_PIF_ISO);
+		regmap_writel(0x0, mmD0_PSOC_BOOT_CONF_D0_PIF_ISO);
 }
 
 static void disable_pcie_error_resp(void)
diff --git a/common/efuse.c b/common/efuse.c
index e7d699d..2023a77 100644
--- a/common/efuse.c
+++ b/common/efuse.c
@@ -45,9 +45,6 @@
 		return 0;
 
 	read_reg = mmEFUSE_READ_CTRL;
-
-	/* TODO: ask suyash to test time of this function */
-	/* TODO: facto 10 of function time ~10mS*/
 	regmap_writel(1, read_reg);
 
 	for (i = 0; i < FUSE_RETRIES; i++)
